rfcs/0000_conducting_research_while_building_a_stable_platform.md

```diff
@@ -0,0 +1,176 @@
+- Start Date: 2025-07-28
+- RFC PR: (leave this empty, it will be filled in after RFC is merged)
+
+# Conducting Research While Building a Stable Platform
+
+[The vision of the Thousand Brains Project is to develop a platform for building AI and robotics applications using the same principles as the human brain.](https://thousandbrainsproject.readme.io/docs/vision-of-the-thousand-brains-project) To achieve this vision, we pursue the platform work while we continue to conduct active research with [open questions still to be answered](https://thousandbrainsproject.readme.io/docs/open-questions) and already [planned future work](https://thousandbrainsproject.readme.io/docs/project-roadmap).
+
+We believe that the research and the platform are essential to our vision. We want to succeed at both.
+
+## Inherent challenge
+
+There lies an inherent challenge to achieving our vision. From the perspective of supply and demand competition in the marketplace, the activities of research and platform building differ in their characteristics. Some of what makes for good research, directly conflicts with what makes for a good platform, and vice versa.
```

Comment:
The "supply and demand" supporting clause seems irrelevant to me, but maybe it's just me 😄. I think the activities of research and platform building simply differ, there's not any particular perspective that makes that so.